conf/pfqueue.conf.defaults

```diff
@@ -17,36 +21,33 @@ redis_every=100
 # general queue configuration
 #
 [queue general]
-workers=4
-has_delayed_queue=enabled
-delayed_queue_batch=100
-delayed_queue_sleep=100
-delayed_queue_workers=1
+weight=4
+workers=1
 
 #
 # pfdhcplistener queue configuration
 #
 [queue pfdhcplistener]
-workers=4
+weight=4
+workers=1
```

Comment:
Just to make sure I get it, will this make sure there is at least 1 dedicated worker for the pfdhcplistener queue ?